src/main.rs

```diff
@@ -346,3 +404,56 @@ async fn promp_task_select(
 
     Ok((project.clone(), task.clone()))
 }
+
+async fn promp_activitie_select(
```

Comment:
Same function as in https://github.com/Xedon/moco_cli/pull/2?
Maybe put it into utils.rs